src/main/java/com/github/mauricioaniche/ck/metric/RFC.java

```diff
@@ -2,48 +2,43 @@
 
 import com.github.mauricioaniche.ck.CKClassResult;
 import com.github.mauricioaniche.ck.CKMethodResult;
+import com.github.mauricioaniche.ck.util.JDTUtils;
 import org.eclipse.jdt.core.dom.*;
 
 import java.util.HashSet;
 import java.util.List;
 
 public class RFC implements CKASTVisitor, ClassLevelMetric, MethodLevelMetric {
 
-	private HashSet<String> methodInvocations = new HashSet<String>();
+	private final HashSet<String> methodInvocations = new HashSet<String>();
 
 	public void visit(MethodInvocation node) {
 		IMethodBinding binding = node.resolveMethodBinding();
 		count(node.getName()  + "/" + arguments(node.arguments()), binding);
```

Comment:
We can probably use the `getMethodFullName` here too!

Reply:
@mauricioaniche `JDTUtils#getMethodFullName` not accept null values.

```java
public void visit(MethodInvocation node) {
	IMethodBinding binding = node.resolveMethodBinding();
	count(node.getName()  + "/" + arguments(node.arguments()), binding);
}
```
Here, sometimes `binding` is null, so I cannot get method full name using `JDTUtils#getMethodFullName`

`JDTUtils#getMethodFullName` not accepts `MethodInvocation` type as argument.

`binding` is always null when method not belongs to class.

```java
visit(ExpressionMethodReference node)
```
have same behavior

Reply:
so, I think you should deal with the null case here, instead of duplicating code.

This is all about getting the name of the method, right? So, suggestion:

```
IMethodBinding binding = node.resolveMethodBinding();
String methodName = binding!=null ? JDTUtils.getMethodFullName(binding) : JDTUtils.getMethodFullName(node);
```

What do you think?

Reply:
I was thinking about this.

I can to create a overload of `JDTUtils#getMethodFullName` that accepts **MethodInvocation** and **ExpressionMethodReference** types. And we can get rid of duplicating code.

Reply:
Makes sense to me! If you need any help, let me know!